src/plugins/plot/inspector/PlotOptionsBrowse.vue

```diff
@@ -83,7 +83,7 @@
                 <div class="grid-cell value">{{ yAxis.rangeMin }}</div>
             </li>
             <li
-                v-if="!yAxis.autoscale && yAxis.rangeMax"
+                v-if="!yAxis.autoscale && yAxis.rangeMax !== ''"
```

Comment:
`!== ''` is redundant here since empty strings are falsy, we can remove it

Reply:
We want to allow rangeMax and rangeMin `0` values. This is why I've specified ''

Reply:
@shefalijoshi 

Is this supposed to be a string or a number?